aten/src/ATen/native/Activation.cpp

```diff
@@ -368,7 +378,17 @@ Tensor softplus_backward(
     const Scalar& threshold,
     const Tensor& output) {
   Tensor grad_input;
-  auto iter = TensorIterator::binary_op(grad_input, grad_output, output);
+  auto iter = TensorIteratorConfig()
```

Comment:
I guess this is the first "ternary op" that uses TensorIterator?

I personally think it would be a little more maintainable just to add a new `TensorIterator::ternary_op()`, since that doesn't add too much cognitive load (we already have 5+ variants of that builder function). And that makes it easy to find all the call-sites of ternary ops, if/when more get added later.

Reply:
Cool, sounds good. I'll add `ternary_op()`.